### PR TITLE
Sql alchemy integration

### PR DESCRIPTION
**Pull request by Goldziher**

### CODE REVIEW

**Review by yudjinn — APPROVED** [review]

File: tests/plugins/sql_alchemy_plugin/test_sql_alchemy_relationships.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    Column("friend_b_id", Integer, ForeignKey("user.id"), primary_key=True),
+)
+
+
+class Pet(Base):
+    id = Column(Integer, primary_key=True)
+    species = Column(Enum(Species))
+    name = Column(String)
+    age = Column(Float)
+    onwer_id = Column(Integer, ForeignKey("user.id"))
```
Comment (yudjinn) [inline tests/plugins/sql_alchemy_plugin/test_sql_alchemy_relationships.py:40]: typo here, should be owner_id

**Review by yudjinn — CHANGES_REQUESTED** [review]

File: tests/openapi/test_parameters.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ def test_create_parameters():
     route_handler = PersonController.get_persons
     parameters = create_parameters(
         route_handler=route_handler,
-        handler_fields=create_function_signature_model(fn=cast(Callable, route_handler.fn)).__fields__,
+        handler_fields=create_function_signature_model(fn=cast(Callable, route_handler.fn), plugins=[]).__fields__,
```
Comment (yudjinn) [inline tests/openapi/test_parameters.py:18]: do you have to send a blank list for plugins?

**Review by yudjinn — APPROVED** [review]

File: tests/openapi/test_parameters.py (hunk 2 above)
Comment (Goldziher) [reply tests/openapi/test_parameters.py:18]: its a required parameter, so an empty list it is.

File: tests/plugins/sql_alchemy_plugin/test_sql_alchemy_relationships.py (hunk 1 above)
Comment (Goldziher) [reply tests/plugins/sql_alchemy_plugin/test_sql_alchemy_relationships.py]: resolved

**Pull request merged**